db/seeds.rb

```diff
@@ -47,9 +47,9 @@
 ronald4 = User.find(4)
 
 # tenants
-Tenant.create!(
+bridge_builders = Tenant.create!(
   location:              "East Timor represent",
-  organization:          "Bridge Builders off the island."
+  organization:          "bridge-builders"
```

Comment:
I would leave this in there. Real organizations will have spaces (and occasionally punctuation). The before_save will ensure that a slug is created.